cvat/apps/iam/rules/memberships.rego

```diff
@@ -92,11 +94,16 @@ allow {
     input.resource.role != organizations.OWNER
 }
 
+# member can leave the organization except cases when member is the owner or maintainer
 allow {
     input.scope == utils.DELETE
     input.resource.is_active
-    utils.is_sandbox
```

Comment:
@kirill-sizov, In case of deleting a membership, the environment will never be a sandbox. https://github.com/opencv/cvat/blob/9148234a159c5d5e193e1fcd785ed7010bf950b7/cvat/apps/iam/permissions.py#L61C29-L61C29